pages/linux/e2fsck.md

```diff
@@ -0,0 +1,15 @@
+# e2fsck
+
+> Check a Linux ext2/ext3/ext4 filesystem. The filesystem should be unmounted at the time the command is run.
+
+- Check filesystem, reporting any damaged blocks:
+
+`e2fsck {{/dev/sdXN}}`
+
+- Check filesystem and automatically repair any damaged blocks:
+
+`e2fsck -p {{/dev/sdXN}}`
+
+- Check filesystem in read only mode:
+
+`e2fsck -c {{/dev/sdXN}}`
```

Comment:
I found this in `e2fsck --help`:
```
 -n                   Make no changes to the filesystem
 -c                   Check for bad blocks and add them to the badblock list
```

`-n` looks a read only mode.

Reply:
@navarroaxel yes, --help flag is showing it but I got this information from man page.

-n     Open the filesystem read-only, and assume an answer of `no' to all questions.  Allows e2fsck to be used non-interactively.
-c     This option causes e2fsck to use badblocks(8) program to do a read-only scan of the device in order to find any bad blocks